docs/faq.md

```diff
@@ -19,15 +19,15 @@ messages from any given date and day-by-day navigation.
 
 ## Why did the archive bot join my room?
 
-Only public Matrix rooms with `shared` or `world_readable` history visibility are
+Only public Matrix rooms with `shared` or `world_readable` [history
+visibility](https://spec.matrix.org/v1.6/client-server-api/#room-history-visibility) are
```

Comment:
Why specify an older version of spec instead of just latest or at least the current 1.7?